src/common/constants/constants.ts

```diff
@@ -1 +1,10 @@
 export const MovieGenres = ['Documentary', 'Comedy', 'Horror', 'Crime']
+
+export type MovieGenresType = (typeof MovieGenres)[number]
```

Comment:
Currently `type MovieGenresType = string`, because for Typescript this is just an array of strings, not a tuple. You can fix it by adding `as const` after the array.